File: data/cgi-bin/modify_comments.py
#!/usr/bin/env python3

import sys
import random

# Read the POST data from stdin
post_data = sys.stdin.read()

# Basic parsing of form data
data = {}
for item in post_data.split("&"):
    key, value = item.split("=")
    data[key] = value

name = data.get("name", "")
message = data.get("message", "")

# Function to scramble words
def scramble_word(word):
    if len(word) <= 1:
        return word
    word_list = list(word)
    random.shuffle(word_list)
    return ''.join(word_list)

# Scramble each word in the message
scrambled_message = " ".join([scramble_word(word) for word in message.split()])

# Print the scrambled message
print(scrambled_message)

File: data/cgi-bin/modify_comments.php
#!/usr/bin/env php
<?php
// Read the POST data
$post_data = file_get_contents("php://stdin");

// Basic parsing of form data
$data = [];
parse_str($post_data, $data);

$name = isset($data['name']) ? $data['name'] : '';
$message = isset($data['message']) ? $data['message'] : '';

// Function to capitalize every letter of a word
function capitalize_word($word) {
    return strtoupper($word); // Capitalize every letter of the word
}

// Capitalize each word in the message
$capitalized_message = implode(" ", array_map("capitalize_word", explode(" ", $message)));

// Output the capitalized message
echo $capitalized_message;
?>
